libs/wai-utilities/src/Network/Wai/Utilities/Server.hs

```diff
@@ -80,8 +79,6 @@ newSettings (Server h p l m t el er) = do
            . setBeforeMainLoop logStart
            . setOnOpen (const $ connStart >> return True)
            . setOnClose (const connEnd)
-           . setOnException onException
```

Comment:
I don't see where you're replacing this behaviour; why remove it? Is it redundant now for some reason?

Reply:
see the commit message of b693f70f51111daa74e99cdc274626d04c6e9292